first_order_model/reconstruction.py

```diff
@@ -35,6 +35,36 @@
 generate_video_visualizations = False
 
 
+def get_size_of_nested_list(list_of_elem):
+    """ helper to get size of nested parameter list """ 
+    count = 0
+    for elem in list_of_elem:
+        if type(elem) == list:  
+            count += get_size_of_nested_list(elem)
+        else:
+            count += 1    
+    return count
+
+
+def get_model_info(log_dir, kp_detector, generator):
```

Comment:
this function is implemented in utils.py now, and can just be called - don't need this or the above helper